docs/en/docs/ecosystem/udf/java-user-defined-function.md

```diff
@@ -107,13 +107,11 @@ The following SimpleDemo will implement a simple function similar to sum, the in
 ```JAVA
 package org.apache.doris.udf.demo;
 
-import org.apache.hadoop.hive.ql.exec.UDAF;
-
 import java.io.DataInputStream;
 import java.io.DataOutputStream;
 import java.io.IOException;
 
-public class SimpleDemo extends UDAF {
```

Comment:
why not extend the UDAF?

Reply:
the is no need't to extend anything, it's meaningless